docs/web/concepts-app-config.mdx

```diff
@@ -0,0 +1,412 @@
+                <map id="small-map" config="small-map" grow="1" margin="4"/>
+                <scalebar config="small-map-scalebar" grow="1" slot="bottom-left" models="#small-map"/>
+                <button grow="1" config="zoom-both-maps"/>
+                <split grow="1" valign="center">
+                    <button grow="1" config="zoom-small-map"/>
+                    <button grow="1" config="zoom-large-map"/>
+                </split>
+                </stack>
+        </panel>
+        <map id="large-map" config="large-map">
+            <iwtm config="iwtm" slot="top-left"/>
+            <compass slot="top-right"/>
+            <web:expand slot="bottom-right" >
+                <web:bookmarks />
+            </web:expand>
+            <web:scale-input slot="bottom-left" />
+            <scalebar slot="bottom-left"/>
+            <map-coordinates slot="context-menu"/>
+        </map>
+    </split>
+</layout>
+```
+
+</TabItem>
+<TabItem value="config">
+
+```json
+{
+    "schemaVersion": "1.0",
+    "items": [
+        {
+            "$type": "geocortex-web-viewer",
+            "id": "default",
+            "onInitialized": [
+                {
+                    "name": "ui.display-notification",
+                    "arguments": {
+                        "message": "Use the I Want To Menu to open up a hidden side panel."
+                    }
+                }
+            ]
+        },
+        {
+            "$type": "map-extension",
+            "id": "small-map"
+        },
+        {
+            "$type": "map-extension",
+            "id": "large-map"
+        },
+        {
+            "id": "iwtm",
+            "items": [
+                {
+                    "iconId": "open",
+                    "title": "Open Hidden Side Panel",
+                    "action": {
+                        "name": "ui.activate",
+                        "arguments": "hidden-panel"
+                    },
+                    "$type": "menu-item"
+                }
+            ],
+            "title": "I Want To Menu",
+            "isLandmark": true,
+            "$type": "menu"
+        },
+        {
+            "$type": "menu-item",
+            "id": "zoom-both-maps",
+            "title": "Zoom Both Maps In",
+            "isEnabled": true,
+            "iconId": "zoom",
+            "action": {
+                "name": "map.zoom-in",
+                "arguments": {
+                    "maps": [
+                        "item://map-extension/small-map",
+                        "item://map-extension/large-map"
+                    ]
+                }
+            }
+        },
+        {
+            "$type": "menu-item",
+            "id": "zoom-small-map",
+            "title": "Zoom Small Map Out",
+            "isEnabled": true,
+            "iconId": "zoom",
+            "action": {
+                "name": "map.zoom-out",
+                "arguments": {
+                    "maps": ["item://map-extension/small-map"]
```

Comment:
I thought it was confusing that the argument was named `maps`, but a singular map could be given. Since both work, I thought it was less confusing to pass an array, which also showcases the possibility of passing multiple maps.

Reply:
OK, that's fair. I wasn't sure if you were aware or not.